build-support/bin/generate_travis_yml.py

```diff
@@ -76,10 +76,10 @@ def all_entries(cls) -> List[Dict[str, str]]:
     # mitigate risk.
     "BOOTSTRAPPED_PEX_KEY_PREFIX=daily/${TRAVIS_BUILD_NUMBER}/${TRAVIS_BUILD_ID}/pants.pex",
     "NATIVE_ENGINE_SO_KEY_PREFIX=monthly/native_engine_so",
-    "PYENV_PY27_VERSION=2.7.18",
-    "PYENV_PY36_VERSION=3.6.10",
-    "PYENV_PY37_VERSION=3.7.7",
-    "PYENV_PY38_VERSION=3.8.3",
+    "MACOS_PYENV_PY27_VERSION=2.7.18",
```

Comment:
Thales pointed out that it was confusing that this is only used by macOS. I agree.